addons/web/static/src/legacy/scss/primary_variables.scss

```diff
@@ -167,3 +167,8 @@ $o-kanban-dashboard-dropdown-complex-gap: 5px !default;
 // Form view
 
 $o-form-view-sheet-max-width: 1140px !default;
+
+// Border-radius
+
+$o-border-radius: 3px !default;
```

Comment:
`.25rem` ? :)
Nothing changes of course, it's just cleaner to use the same units... at least within the same variables stack